deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -230,8 +230,7 @@ spec:
             spec:
               containers:
               - args:
-                - --zap-level=debug
-                - --zap-encoder=console
+                - --debugLogging
```

Comment:
a reminder to myself: this will now be needed to update in the render template script to turn off debug logging in release.